saleor/graphql/meta/resolvers.py

```diff
@@ -19,6 +19,8 @@ def resolve_object_with_metadata_type(instance: ModelWithMetadata):
     from ..checkout import types as checkout_types
     from ..order import types as order_types
     from ..product import types as product_types
+    from ..invoice import types as invoice_types
+    from ...invoice import models as invoice_models
```

Comment:
We can import model outside function.

Reply:
Cannot do, it is done that way (like other impoets here) so we don't fall into circular import.